ha/util/conf_store.py

```diff
@@ -41,13 +44,23 @@ def _get_data_pods(self, index):
         Get machine ids for data pods: returns list of machine ids.
         """
         machine_ids = []
+        keys = Conf.search(index, NODE_CONST, SERVICE_CONST, Const.SERVICE_MOTR_IO.value)
```

Comment:
I think, keys can be renamed to some better understandabel variable. Also, can you please add one sample output here that comes from serach API?

Reply:
Done both